### PR TITLE
feat: add helm

### PR DESCRIPTION
**Pull request by charlax**

### CODE REVIEW

File: helm-chart/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.4.5"
```
Comment (tcrasset) [inline helm-chart/Chart.yaml:15]: This is going to be difficult to maintain

File: helm-chart/templates/api-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+metadata:
+  name: api
+  labels:
+    {{- include "avatar.labels" . | nindent 4 }}
+type: Opaque
+
+data:
+  pepper: {{ .Values.api.pepper | b64enc | quote }}
+  authjwt_secret_key: {{ .Values.api.authjwtSecretKey | b64enc | quote }}
+  file_encryption_key: {{ .Values.api.fileEncryptionKey | b64enc | quote }}
```
Comment (tcrasset) [inline helm-chart/templates/api-secrets.yaml:12]: `b64enc` looks handy! I guess you doubled checked that you indeed need to b64 encode the encryption key, which is already in base64 ?

File: helm-chart/templates/cron-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  template:
+    metadata:
+      labels:
+        {{- include "avatar.labels" . | nindent 8 }}
+        app.kubernetes.io/component: cron
+    spec:
+      containers:
+        - name: avatar-cron
+          image: quay.io/octopize/api:{{ .Values.avatarVersion }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
```
Comment (tcrasset) [inline helm-chart/templates/cron-deployment.yaml:22]: Nice!

**Review by tcrasset — APPROVED** [review]
Looks good !

File: helm-chart/Chart.yaml (hunk 1 above)
Comment (charlax) [reply helm-chart/Chart.yaml:15]: Agreed.

File: helm-chart/templates/api-secrets.yaml (hunk 2 above)
Comment (charlax) [reply helm-chart/templates/api-secrets.yaml:12]: yes :)

**Pull request merged**